<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QMainWindow" name="VentanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(21, 32, 43);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Binicio">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(20, 185, 255);
	border-color: rgb(255, 255, 255);
color : rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Inicio</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>571</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget{
background-color: rgb(21, 32, 43);
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QListWidget" name="ListaPublicaciones">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>611</width>
        <height>661</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
background-color: rgb(21, 32, 43);
color : rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>541</width>
        <height>141</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/new/prefix1/fondo-codigo-binario_87720-794.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/new/prefix1/USR.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="M_nombre">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="M_Follower">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>190</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color: rgb(255, 255, 255);
}
</string>
      </property>
      <property name="text">
       <string>Comentarios: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="M_Publicacion">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color: rgb(255, 255, 255);
}
</string>
      </property>
      <property name="text">
       <string>Publicaciones:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="M_Comentarios">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color: rgb(255, 255, 255);
}
</string>
      </property>
      <property name="text">
       <string>Seguidores:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color: rgb(255, 255, 255);
}
</string>
      </property>
      <property name="text">
       <string>Usuario:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>531</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/new/prefix1/campa_a_cruzcampo_1.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>21</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <zorder>M_nombre</zorder>
     <zorder>M_Follower</zorder>
     <zorder>label_4</zorder>
     <zorder>M_Publicacion</zorder>
     <zorder>label_3</zorder>
     <zorder>M_Comentarios</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>label_14</zorder>
     <zorder>pushButton_7</zorder>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(20, 185, 255);
	border-color: rgb(255, 255, 255);
color : rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Explorar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(20, 185, 255);
color : rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Notificaciones</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-color: rgb(255, 255, 255);
	background-color: rgb(20, 185, 255);
color : rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Mensajes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(20, 185, 255);
	border-color: rgb(255, 255, 255);
color : rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Guardados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(20, 185, 255);
color : rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(20, 185, 255);
color : rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>publicar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/Iinin.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>55</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/IMen.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/Iguarda.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/Ipub.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/usua.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/IBUS.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/INoti.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Binicio_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(20, 185, 255);
color : rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>TOP_PUB</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ListaUsuarios">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>80</y>
      <width>161</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{

color: rgb(255, 255, 255);
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="L_BUSCAR">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>35</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>Buscar:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
